lib/src/HttpControllersRouter.h

```diff
@@ -113,6 +114,7 @@ class HttpControllersRouter : public trantor::NonCopyable
     const std::vector<
         std::function<void(const HttpRequestPtr &, const HttpResponsePtr &)>>
         &postHandlingAdvices_;
+    optional<DefaultHandler> defaultHandler_;
```

Comment:
I think DefaultHandler type is enough here, the bool operator is overloaded in the std::function<> template.